README.md

```diff
@@ -28,19 +28,10 @@ cd <dir>
 git checkout <tag/branch>
 ```
 
-Use git to initialize and clone submodule Galaxy:
+Install Galaxy and dependencies
 
 ```sh
-git submodule init
-git submodule update
-```
-
-Create database and install dependencies required by Galaxy:
-
-```sh
-cd galaxy
-sh install_dependencies.sh 
-sh create_db.sh
+sh install-galaxy.sh
```

Comment:
when building at OSC on webtest02, should I first load a specific Python version? For example I just started with the default python

```
efranz@webtest02:~$ python --version
Python 2.7.5
```

But there is also 

```
efranz@webtest02:~$ scl enable rh-python35 -- python --version
Python 3.5.1
```

And we can install https://www.softwarecollections.org/en/scls/rhscl/rh-python36/

Reply:
Oops I was installing on webtest02. I forgot this line in the README:

>The Install process runs on the **login node**

Yeah that makes sense. In that case, which python module should be used on the Owens login node?